### PR TITLE
feat(cli): added service component management commands

### PR DESCRIPTION
**Pull request by jsam**
This PR introduces service sub-command which is used for service component management.

Introduced commands are the following:

```
Usage: renku service [OPTIONS] COMMAND [ARGS]...

  Manage service components.

Options:
  -e, --env FILE  Path to the .env file.
  -h, --help      Show this message and exit.

Commands:
  api        Start service JSON-RPC API in active shell session.
  scheduler  Start service scheduler in active shell session.
  worker     Start service worker in active shell session.
```

Followup:
* Add commands `renku service up` and `renku service down` to control all components with *local* daemon mode

closes: #1751 

/deploy

### CODE REVIEW

**Comment by rokroskar** [comment]
If I install renku without the service component, it crashes the CLI: 

```
renku init test-repo
Traceback (most recent call last):
  File "/Users/rok/.local/bin/renku", line 33, in <module>
    sys.exit(load_entry_point('renku', 'console_scripts', 'renku')())
  File "/Users/rok/.local/bin/renku", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/rok/Projects/renku-core/renku-python/renku/cli/__init__.py", line 89, in <module>
    from renku.cli.service import service
  File "/Users/rok/Projects/renku-core/renku-python/renku/cli/service.py", line 23, in <module>
    from dotenv import load_dotenv
ModuleNotFoundError: No module named 'dotenv'
```

**Comment by jsam** [comment]
> If I install renku without the service component, it crashes the CLI:
> 
> ```
> renku init test-repo
> Traceback (most recent call last):
>   File "/Users/rok/.local/bin/renku", line 33, in <module>
>     sys.exit(load_entry_point('renku', 'console_scripts', 'renku')())
>   File "/Users/rok/.local/bin/renku", line 25, in importlib_load_entry_point
>     return next(matches).load()
>   File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/metadata.py", line 77, in load
>     module = import_module(match.group('module'))
>   File "/usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
>     return _bootstrap._gcd_import(name[level:], package, level)
>   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
>   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
>   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
>   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
>   File "<frozen importlib._bootstrap_external>", line 790, in exec_module
>   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
>   File "/Users/rok/Projects/renku-core/renku-python/renku/cli/__init__.py", line 89, in <module>
>     from renku.cli.service import service
>   File "/Users/rok/Projects/renku-core/renku-python/renku/cli/service.py", line 23, in <module>
>     from dotenv import load_dotenv
> ModuleNotFoundError: No module named 'dotenv'
> ```

Should be fixed now.

File: Dockerfile.svc [hunk 1]
```diff
@@ -35,13 +35,5 @@ USER shuhitsu
...
-    "gunicorn \
-    renku.service.entrypoint:app \
-    -b 0.0.0.0:8080 \
-    --timeout 600 \
-    --workers $RENKU_SVC_NUM_WORKERS \
-    --worker-class gthread \
-    --threads $RENKU_SVC_NUM_THREADS" \
-]
+
+CMD ["renku", "service", "api"]
```
Comment (rokroskar) [inline Dockerfile.svc:39]: Just a suggestion - if the entrypoint is `["renku", "service"]` then the invocation of the container is simply e.g.

`docker run renku/renku-core api` or `docker run renku/renku-core worker` and in the chart we would just have e.g.

```
args: api
```

**Comment by rokroskar** [comment]
The service imports need to be conditional somehow - I'm still getting errors like

```
 rok  /tmp  renku
Traceback (most recent call last):
  File "/Users/rok/.local/bin/renku", line 5, in <module>
    from renku.cli import cli
  File "/Users/rok/.local/pipx/venvs/renku/lib/python3.9/site-packages/renku/cli/__init__.py", line 89, in <module>
    from renku.cli.service import service
  File "/Users/rok/.local/pipx/venvs/renku/lib/python3.9/site-packages/renku/cli/service.py", line 25, in <module>
    from renku.service.jobs.queues import QUEUES
  File "/Users/rok/.local/pipx/venvs/renku/lib/python3.9/site-packages/renku/service/jobs/queues.py", line 21, in <module>
    from rq import Queue
ModuleNotFoundError: No module named 'rq'
```

**Review by Panaetius — CHANGES_REQUESTED** [review]

**Review by Panaetius — APPROVED** [review]
Thank you :tada:

**Pull request merged**